src/registry.ts

```diff
@@ -67,13 +53,7 @@ export async function lookupRegistry(
   throw new Error(`unexpected response status code ${response.status}`);
```

Comment:
I don't know, cc @peterjanbrone

Reply:
@kwypchlo what is more idiomatic, throwing or returning null?

Reply:
Yeah I decided to throw here and return null because this is basically a sanity check to catch developer error.
That or the siad API changed, we basically do not expect to throw this error ever and I wanted to avoid the `try-catch` on the caller side if we want to stay consistent in the expected case.

It's kind of like `build.Critical` or `panic` in siad, at least that was my intention.

Reply:
@peterjanbrone @m-cat SDK should probably abstract that away, otherwise we will end up with people having to check if the response is not null AND also writing a try-catch.